client/src/components/ElementEditor/Element.js

```diff
@@ -503,6 +519,10 @@ Element.propTypes = {
   onDragOver: PropTypes.func, // eslint-disable-line react/no-unused-prop-types
   onDragEnd: PropTypes.func, // eslint-disable-line react/no-unused-prop-types
   onDragStart: PropTypes.func, // eslint-disable-line react/no-unused-prop-types
+  saveElement: PropTypes.bool.isRequired,
+  onBeforeSubmitForm: PropTypes.func.isRequired, // eslint-disable-line react/no-unused-prop-types
+  onAfterSubmitResponse: PropTypes.func.isRequired,
+  increment: PropTypes.number.isRequired,
```

Comment:
```suggestion
  // Used to ensure form gets re-rendered on submission so it can be submitted again if there's validation errors
  increment: PropTypes.number.isRequired,
```
The name isn't self-explanatory, so we should explain what this is here for.

Reply:
Updated